composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=5.5",
-        "elasticsearch/elasticsearch": "~2.0"
+        "elasticsearch/elasticsearch": "dev-2.0"
```

Comment:
we must wait for a new release including the `setConnectionParams()` ClientBuilder method